styles/style.css

```diff
@@ -0,0 +1,1088 @@
+}
+button:has(i) {
+    border: none;
+    background: none;
+    width: 2rem;
+}
+.remove {
+    content: url("../assets/icons/delete.svg");
+}
+.add {
+    content: url("../assets/icons/arrowRight.svg");
+}
+.next {
+    width: 3rem;
+    height: 3rem;
+    right: 0.5rem;
+    background: white;
+    border-radius: 2px;
+    box-shadow: 0px 1px 0px 0px #0000001A;
+    cursor: pointer;
+}
+.next i {
+    content: url("../assets/icons/chevronRightBlue.svg");
+}
+/* einde iconen */
+button.btn {
+    background: orange;
+    border: none;
+    color: white;
+    border-radius: 2px;
+    padding: 0.75rem 1rem;
+    width: 100%;
+    font-size: 1rem;
+    cursor: pointer;
+}
+.w100 {
+    width: 100%;
+}
+/* EINDE STANDAARD STYLES */
+
+
+
+/* HEADER */
+/* menu nav */
+header nav {
+    width: 100vw;
+    height: 3rem;
+    padding: 0.5rem 12rem; 
+    background-color: var(--mainColor);
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    overflow: hidden;
+    z-index: 999;
+}
+header nav .logo{
+    width: 90px; 
+}
+header nav .logo img{
+    width: 100%; 
+    height: 100%;
+}
+
+header nav div {
+    display: flex;
+    flex-direction: row;
+    color: white;
+    align-items: center;
+}
+header nav ul {
+    display: flex;
+    list-style: none;
+}
+header nav ul li a {
+    color: white;
+    text-decoration: none;
+    padding: .95em 1rem;
+}
+header nav ul li a:hover,
+header nav ul li a:active,
+button.btn:hover {
+    background-color: var(--hoverOrangeColor);
+}
+header nav ul li ul{
+    display: none;
+}
+
+.menuIcons a {
+    background: none;
+    border: none;
+    color: white;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+}
+.menuIcons a span {
+    margin-left: 0.5rem;
+}
+.menuIcons .mobileMenu {
+    display: none;
+}
+/* einde menu nav */
+
+
+
+/* header image */
+section.headerImage {
+    width: 100%;
+    margin-top: 3rem;
+    border-bottom: 2px solid var(--mainColor);
+}
+.objectImages {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: 1fr 1fr;
+    width: 100%;
+    gap: .25rem; 
+}
+.mainImage {
+    grid-column: 1 / span 2; /* start gridlijn 1 en eindigt bij 2 */
+    grid-row: span 2; /* eindigt bij rij 2*/
+    height: 100%;
+}
+.subImage {
+    background-color: var(--mainColor);
+}
+.objectImages a {
+    display: flex;
+    width: 100%;
+    height: 100%;
+}
+.objectImages a img {
+    width: 100%;
+    object-fit: cover;
+}
+
+.detailList {
+    color: var(--thirdFontColor);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+.detailList ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 1.5em;
+    list-style: none;
+}
+.detailList a {
+    padding: 1rem 0;
+}
+.detailList span,
+.afmetingen span {
+    color: var(--secondFontColor);
+}
+
+.detailList ul li a {
+    display: flex;
+    gap: 0.35rem;
```

Comment:
Deze gap is wel heel minimalistisch waardoor ik me af vraag of je `grid` nodig hebt of dat dit een styling issue geeft.